### PR TITLE
Dev/hshami/no sudo

### PR DESCRIPTION
**Pull request by mhshami01**
drop sudo requirement

### CODE REVIEW

**Review by marianan — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -29,40 +29,32 @@ The following table compares the requirements to run your solution on the IoT Ed
     pip install --upgrade iotedgehubdev
     ```
     **Note**: Please install iotedgehubdev to **root** on Linux/macOS (*Don't use '--user' option in the 'pip install' command*).
+5. (Linux / MacOS only)
```
Comment (marianan) [inline README.md:32]: >Linux / MacOS only) [](http://example.com/codeflow?start=4&length=19)

Is there anything to do for Windows?

File: README.md [hunk 2]
```diff
@@ -29,40 +29,32 @@ The following table compares the requirements to run your solution on the IoT Ed
     pip install --upgrade iotedgehubdev
     ```
     **Note**: Please install iotedgehubdev to **root** on Linux/macOS (*Don't use '--user' option in the 'pip install' command*).
```
Comment (marianan) [inline README.md:31]: >**Note**: [](http://example.com/codeflow?start=4&length=9)

does this note still apply?

File: README.md (hunk 1 above)
Comment (mhshami01) [reply README.md]: Nothing special for Windows

File: README.md (hunk 2 above)
Comment (mhshami01) [reply README.md]: It does not - Thank you!

**Pull request merged**